.travis.yml

```diff
@@ -1,4 +1,12 @@
 language: csharp
-sudo: false
+sudo: required
```

Comment:
+1. We're pushing folks towards the container infrastructure as it's much faster and more reliable. Installing libuv locally without sudo should work fine.